plugins/modules/vcenter_folder.py

```diff
@@ -180,32 +179,28 @@ def ensure(self):
                     parent_folder_parts = parent_folder.strip('/').split('/')
                     p_folder_obj = None
                     for part in parent_folder_parts:
-                        part_folder_obj = self.get_folder(datacenter_name=datacenter_name,
```

Comment:
Hi @lydiym, thank you for the PR. I'm not sure I understand why you remove the `datacenter_name` key?

Reply:
Just because it is unused in ensure/get_folder dirrectly. But indirrectly through datacenter_folder_type still yes! Now it is unnesesery to test datacenter in get_folder, like that was in previous version.